ASTProcessor/build.gradle.kts

```diff
@@ -19,10 +19,18 @@ repositories {
 dependencies {
 	implementation(libs.springboot.starter.web)
 	implementation(libs.springdoc.openapi)
+	implementation(libs.antlr.runtime)
+	compileOnly(libs.lombok)
+	annotationProcessor(libs.lombok)
```

Comment:
You could order the dependencies so that they are all something like this:

implementations at the beginning
initials for `annotation`
initials for `test`

So that you understand me, I'll give you an example:

![image](https://github.com/user-attachments/assets/c029ffb5-7c2d-4c4d-a2dd-df7da6c4aad3)

Reply:
Done